dcos/dcos-ui
# DCOS-58366: Nodes: Filter field - some special symbols are breaking the field

Make sure the page is not broken when entering certain symbols.

Closes https://jira.mesosphere.com/browse/DCOS-58366

## Testing
1. Go to nodes page.
2. Type "%" in the filter.
3. Verify that the page does not break.
4. Verify that the added test makes sense.

## Trade-offs
Like Slavena noticed, the field is red for some symbols. I will investigate if this is intentional.

## Dependencies
None.

## Screenshots
### Before
![Peek 2019-09-05 13-20](https://user-images.githubusercontent.com/40791275/64334006-5c51cf00-cfe0-11e9-92c3-f6c7d891f902.gif)

### After
![Peek 2019-09-05 13-22](https://user-images.githubusercontent.com/40791275/64334005-5c51cf00-cfe0-11e9-8b62-47d83850448b.gif)
